packages/nx/src/native/native-file-cache-location.ts

```diff
@@ -0,0 +1,19 @@
+import { existsSync, removeSync, rmSync } from 'fs-extra';
+import { tmpdir } from 'os';
+import { join } from 'path';
+
+export const nativeFileCacheLocation = join(tmpdir(), 'nx-native-cache');
```

Comment:
We don't need a separate file for this.

This should be relative to the `cacheDir`